cmd/app_test.go

```diff
@@ -238,6 +238,11 @@ func TestRunAPI(t *testing.T) {
 		var resp *http.Response
 		var body string
 		var errs []error
+		resp, body, errs = gorequest.New().
+			Get("http://127.0.0.1:9090/robots.txt").End()
+		require.Len(t, errs, 0)
+		require.Equal(t, http.StatusOK, resp.StatusCode)
+		require.Contains(t, body, "robotstxt.org")
```

Comment:
this text doesn't appear in robots.txt - why is it expected?

Reply:
It actually is in robots.txt. The PR shows a rename but you can find it 
https://github.com/data-preservation-programs/singularity/blob/main/dashboard/public/robots.txt